### PR TITLE
deletes modular items but this time i don't get cucked

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
it's me again
@ me if it breaks

## Why It's Good For The Game
modular_citadel man bad and you should feel ashamed for adding to it

## Changelog
:cl:
code: modular citadel loses some files
/:cl:

### CODE REVIEW

**Review by Ghommie — APPROVED** [review]

**Comment by ghost** [comment]
this has been "open" for more than 24 hours but i had to remake the pr because the shockcollar pr got merged and fucked with my pr

**Comment by ghost** [comment]
after like 30 minutes of fucking around with git bash i finally realized i was make changes on the wrong branch 😂

**Comment by Ghommie** [comment]
More EoL conflicts because the master .dme is now properly LF.

File: code/init.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-//init file stolen from hippie
+//init file stolen from hippie but the real question is why
 
 /proc/cit_initialize()
```
Comment (Ghommie) [inline code/init.dm]: Hold on, just move the contents to `game/world.dm` and remove this proc and the file. It's just mentors and loadout items after all.

**Review by Ghommie — CHANGES_REQUESTED** [review]

File: code/init.dm (hunk 1 above)
Comment (ghost) [reply code/init.dm]: i moved it to the same place as admin/donator loadout loading so it should be fine now

**Review by Ghommie — APPROVED** [review]

**Comment by ghost** [comment]
it's not a real shit pr unless it's webedited and has 4x as many commits as changes

**Pull request merged**